### ISSUE TITLE
Sonographic renal salt and pepper pattern

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7361]
**Preferred term label:**
Sonographic renal salt and pepper pattern
**Synonyms**

**Definition (free text, please give PubMed ID)**
Heterogeneous, increased parenchymal echogenicity. This imaging appearance is caused by multiple tiny cysts smaller than or just approaching the size necessary for detection, which disrupt the echo pattern without being clearly distinguishable, and appear as white "grains" on renal ultrasound, mainly in a subcapsular location.

PMID:30599104
PMID:22034641

Comment: See Figure 2 of PMID:22034641 and Figure 11 of PMID:30599104.

**Parent term (use hpo.jax.org/app)**
[Abnormal renal echogenicity HP:0033130](https://hpo.jax.org/app/browse/term/HP:0033130)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Ascertained by 1. renal ultrasonography, 2. prenatal renal ultrasonography

**Your nano-attribution (ORCID)**

**Comment by monicacecilia** [comment]
- Would like validation from an ultrasound expert.
- Do we need synonyms? 
  - Consider inclusion of a medical term describing the phenotype as ‘preferred’ name, instead of the more colloquial ‘salt and pepper’ 
  - E.g., stippled; stippled pattern;   
- _The "salt and pepper" pattern on renal ultrasound in a group of children with molecular-proven diagnosis of ciliopathy-related renal diseases_ - https://pubmed.ncbi.nlm.nih.gov/32040628/

**Comment by pnrobinson** [comment]
The term is valid, we can return to add synonyms during a workshop!
Adding new term: 
- ID: HP:6000368
- label: Sonographic renal salt and pepper pattern
- parents: [Abnormal renal echogenicity (HP:0033130)](https://hpo.jax.org/app/browse/term/HP:0033130)
- pmid: PMID:30599104|PMID:22034641
- def: Heterogeneous, increased parenchymal echogenicity. This imaging appearance is caused by multiple tiny cysts smaller than or just approaching the size necessary for detection, which disrupt the echo pattern without being clearly distinguishable, and appear as white grains on renal ultrasound, mainly in a subcapsular location.
- comment: See Figure 2 of PMID:22034641 and Figure 11 of PMID:30599104.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7361
- synonyms:
